### DISCUSSION TITLE
Cannot load existing SpatieMediaLibraryFileUpload image when editing

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#8120] [Help] [answered]
In my multitenant app, SpatieMediaLibraryImageColumn images are loadable in a resource
![image](https://github.com/filamentphp/filament/assets/946010/7de4d9c2-9f10-44f2-8e0b-67c8e38e173a)

But when I try to edit a record, the image loads forever
![image](https://github.com/filamentphp/filament/assets/946010/b086f701-0440-46de-99dc-5f35cfd6ff06)

Looking inside the Chrome dev tools, I see a `CORS error`

The images are from https://centralexample.com/images/1.png and the filament admin is on https://tenant.centralexample.com

Any idea how to make filament load and edit an image with relative path?

**Comment by danharrin** [comment, marked as answer]
If there's a CORS error, you need to fix the CORS policy to allow it, it's not a Filament issue

**Reply by bilogic** [reply]
Ok, thanks let me check.

Initially, my thoughts were that, since multitenancy and SpatieMediaLibraryFileUpload are both first party code, CORS should be addressed or highlighted somewhere in the docs. But I think the multitenancy doesn't support for tenant domains out of the box.

**Reply by danharrin** [reply]
How can we highlight CORS in the docs? It is a case by case basis really. The error tells you that CORS is the problem so it's not like you have to search hard to find that out.

**Reply by bilogic** [reply]
I was trying to explain why I asked this in the first place. But since filament's multitenancy doesn't support tenant domains, it will never face CORS issues, so no need to document or address it. Thanks.
